### ISSUE TITLE
Binding errors to Value and Brush property

### ISSUE THREAD

**Issue by sinatrocious** [Projektanker/Icons.Avalonia#112]
## Describe the bug

After updating from 8.2.0 to 9.0.1 every `Icon` produces 2 errors in the "Output" window:

```
[Binding]Error in binding to 'Projektanker.Icons.Avalonia.IconImage'.'Value': 'Null value in expression '{empty}' at ''.'(IconImage #49591885)
[Binding]Error in binding to 'Projektanker.Icons.Avalonia.IconImage'.'Brush': 'Null value in expression '{empty}' at ''.'(IconImage #49591885)
```

I guess it's due to [this change](https://github.com/Projektanker/Icons.Avalonia/blob/3607f5d15b491db21f752018ae6c9aee53498e21/CHANGELOG.md?plain=1#L22).

## To reproduce

1. Create new avalonia C# application.
2. Add nugets `Projektanker.Icons.Avalonia`, `Projektanker.Icons.Avalonia.FontAwesome`.
3. Register icon provider in `Main`:

```c#
IconProvider.Current.Register<FontAwesomeIconProvider>();
```

4. Add icon to any view:

```xaml
<UserControl ... xmlns:i="clr-namespace:Projektanker.Icons.Avalonia;assembly=Projektanker.Icons.Avalonia">
    <i:Icon Value="fas fa-user"/>
</UserControl>
```
5. Start application and observe debug "Output" window.
6. Optionally add more icons, each will produce two lines with binding errors.

## Expected behavior

There should be no errors.

**Comment by just-seba** [comment]
There seems to be a bug in Avalonia itself. See: https://github.com/AvaloniaUI/Avalonia.Xaml.Behaviors/issues/56

I'll keep this open until the bug is resolved.

**Comment by sinatrocious** [comment]
@just-seba, ok thanks. I am forced to use an older version then, just can't stand the spam.

**Comment by just-seba** [comment]
Should be fixed with [version 9.1.1](https://www.nuget.org/packages/Projektanker.Icons.Avalonia/9.1.1).
